### PR TITLE
Add retries to push_to_queue producer

### PR DESCRIPTION
**Pull request by rocha**
Don't fail immediately when trying connecting to the Broker. Retry a few times with some small pause.

### CODE REVIEW

**Comment by jarv** [comment]
looks good

**Comment by rocha** [comment]
@cpennington can you do a sanity check? This is a quick hack to help @jarv with the certificate problems he is having.

File: queue/producer.py [hunk 1]
```diff
@@ -1,31 +1,56 @@
...
+    retries = 0
+    while True:
+        try:
+            connection = pika.BlockingConnection(parameters)
+        except pika.exceptions.AMQPConnectionError as e:
+            retries += 1
+            if retries >= MAX_RETRIES:
+                raise e
+            time.sleep(RETRY_TIMEOUT)
+            continue
```
Comment (cpennington) [inline queue/producer.py:38]: Don't need to `continue` here, since it's at the end of the block anyway

**Comment by cpennington** [comment]
For a quick hack, I'd have liked to see the hack unobscured by the (nice to have) whitespace changes. :+1: nonetheless.

File: queue/producer.py (hunk 1 above)
Comment (rocha) [inline queue/producer.py:38]: I added it to make the end of the block explicit (the same thing as using the else in the try statement, it is not necessary).

**Pull request merged**